A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -616,6 +616,20 @@ var A320_Neo_UpperECAM;
                     {
                         name: "CONFIG",
                         messages: [
+                            {
+                                id: "config_slats",
+                                message: "",
+                                level: 3,
+                                flightPhasesInhib: [5, 6, 7, 8, 10], // TODO
```

Comment:
```suggestion
                                flightPhasesInhib: [5, 6, 7, 8], // TODO
```
You can drop the 10 here and below according to my docs, but it's not a big deal.